loleaflet/src/control/Toolbar.js

```diff
@@ -609,6 +611,9 @@ L.Map.include({
 				if (!calledFromMenu) {
 					localStorage.setItem('WSDWelcomeVersion', app.socket.WSDServer.Version);
 				}
+				map.welcomeClosed = true;
+				// call Map.Feedback.js or sent a onMessage value?
+				// app.socket._onMessage('feedback-later');
```

Comment:
@hcvcastro (cc: @eszkadev ) as the commit message explains I think we need to avoid showing the feedback iframe while the welcome dialog is opened, specially because they both appear without being triggered by the user. Bottom line is I couldn't figure out how to do that... so help would be welcomed

Reply:
feel free to remove this commit and add your own

Reply:
@pedropintosilva somtehing like this should be fine:

`var checkWelcomeAndRunFeedback = function() {
   if (welcome.exists())
         setTimeout(checkWelcomeAndRunFeedback, 3000); // wait 3s more
   else
         runFeedback();
}

setTimeout(checkWelcomeAndRunFeedback, initialDelay)`

Reply:
@hcvcastro : I'm getting a bit nervous with the time passing (and the probably of dealing again and again with some conflicts) of this PR so, I end up removing the WIP commit and only adjusting the timer value. And let's merge as soon as we get the green again. The additional commit (to prevent welcome and feedback dialog appearing at the some time) can be added later and, at least for now, the longer timer should already make for a better experience.

Reply:
> @pedropintosilva somtehing like this should be fine:
> 
> ```
> var checkWelcomeAndRunFeedback = function() {
>    if (welcome.exists())
>          setTimeout(checkWelcomeAndRunFeedback, 3000); // wait 3s more
>    else
>          runFeedback();
> }
> 
> setTimeout(checkWelcomeAndRunFeedback, initialDelay)
> ```

@eszkadev the time should always be `30000` and what we should do is not show feedback If welcome.exists(). Meaning: not delaying; instead really not showing at all until the close of welcome is triggered via vex `_showWelcomeDialogVex`:`beforeClose`. Just upon being close we start a timer. This will ensure that a person can take as much time as desired to read the welcome dialog and check all the various links that are in it.

Reply:
but as I said before let's add that in in a different PR, so this one can get merged without further delay

Reply:
Thanks @eszkadev for the guidance I have added a new PR : https://github.com/CollaboraOnline/online/pull/2969